### PR TITLE
[VL-146] - Add section about testable use case tc-send-01-bis

### PR DESCRIPTION
**Pull request by kin0992**
Add information about TC-SEND-01bis use case.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,22 @@ Create a `NotificationRequest` providing two documents: the act to notify and th
...
+
+1. Request an upload slot
+2. Consume the upload slot uploading the document to notify (use the information provided in the previous request).
+3. Create a notification request providing the following information:
+    - `physicalCommunicationType` filled with `REGISTERED_LETTER_890` value
+    - the `recipients` field providing:
+        - `taxId`
+        - `digitalDomicile`
+        - `physicalAddress`
+        - `payment` referencing one file uploaded previously
```
Comment (lordkada) [inline README.md:50]: The payment field shouldn't be filled here; the use case is about sending a document without any payment, right?

File: README.md [hunk 2]
```diff
@@ -36,6 +36,22 @@ Create a `NotificationRequest` providing two documents: the act to notify and th
...
+    - the `documents` field providing the file uploaded previously
+
+__Note__: this use case differs from the previous one in that the `documents` field contains only one file
+(the act to notify) and doesn't contain the pagoPA payment.
```
Comment (datalek) [inline README.md:54]:
```suggestion
__Note__: This use case is different from the previous one in that a pagoPA payment is not involved. For this reason this use case requests at least **one** document.
```

File: README.md [hunk 3]
```diff
@@ -36,6 +36,22 @@ Create a `NotificationRequest` providing two documents: the act to notify and th
       - `payment` referencing one file uploaded previously
    - the `documents` field providing one file uploaded previously
 
+### TC-SEND-01bis
+Create a `NotificationRequest` providing only the act to notify.
+
+1. Request an upload slot
+2. Consume the upload slot uploading the document to notify (use the information provided in the previous request).
```
Comment (lordkada) [inline README.md:43]: _use the information provided in the previous request_: you mean "..in the previous response"?

**Comment by lordkada** [comment]
I suggest a further simplification (I also realigned the TC-SEND-01 case too). It looks to me a little bit more straight to the point.

### TC-SEND-01
Create a `NotificationRequest` providing two documents, the act to be notified and the pagoPA payment.

1. Request two upload slots.
2. Upload the two documents, act and payment, using the returned slots.
3. Create a notification request carrying the following information:
   - `physicalCommunicationType` filled with `REGISTERED_LETTER_890` value
   - the `recipients` field containing:
      - `taxId`
      - `digitalDomicile`
      - `physicalAddress`
      - `payment` referencing the payment file previously uploaded
   - the `documents` field providing the act file previously uploaded

### TC-SEND-01bis
Create a `NotificationRequest` providing the act to be notified only (no payments are involved).

1. Request an upload slot.
2. Upload the act document using the returned slot.
3. Create a notification request carrying the following information:
    - `physicalCommunicationType` filled with `REGISTERED_LETTER_890` value
    - the `recipients` field containing:
        - `taxId`
        - `digitalDomicile`
        - `physicalAddress`
    - the `documents` field providing the act file previously uploaded

**Comment by lordkada** [comment]
For the sake of clarity, I've included the above review proposal in a separate [PR](https://github.com/pagopa/pn-local-emulator/pull/76)

**Review by lordkada — APPROVED** [review]

**Pull request merged**